content/updates/2017-07-18-release-notes.md

```diff
@@ -15,12 +15,12 @@ Curious what’s new that you might find helpful as a cloud.gov user? Here are h
 If you have .NET applications you’re interested in running on cloud.gov, the updated [.NET Core buildpack](https://docs.cloudfoundry.org/buildpacks/dotnet-core/index.html) supports [.NET Core 2 Preview (currently in alpha)](https://blogs.msdn.microsoft.com/dotnet/2017/06/28/announcing-net-core-2-0-preview-2/). It’s ready for you to prototype with your .NET applications.
 
 ### Platform releases
-We upgraded the Cloud Foundry deployment to [v267](https://github.com/cloudfoundry/cf-release/releases/tag/v264).
+We upgraded the Cloud Foundry deployment to [v267](https://github.com/cloudfoundry/cf-release/releases/tag/v267).
 
 You should restage your application to incorporate fixes in the base filesystem and ensure you’re running the most recent language version supported by your buildpack.
 
 #### Additional upgrades
-* [RootFS cflinuxfs2 1.134.0](https://github.com/cloudfoundry/cflinuxfs2/releases/tag/1.138.0), which addresses this security vulnerability:
+* [RootFS cflinuxfs2 1.134.0](https://github.com/cloudfoundry/cflinuxfs2/releases/tag/1.134.0), which addresses this security vulnerability:
```

Comment:
Not sure if this was supposed to be corrected to 1.134.0 or if the text was supposed to be corrected to to 1.138.0